packages/firestore/test/integration/api/count.test.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+  countQuery,
+  getAggregateFromServerDirect,
+  query
+} from '../util/firebase_export';
+import { apiDescribe, withTestCollection } from '../util/helpers';
+
```

Comment:
both tests will fail as the count API internal logic is not implemented yet, and only returning 42 as dummy count.